### ISSUE TITLE
Configurable storage for VM & resources

### ISSUE THREAD

**Issue by jorycz** [yep/virtualOS#41]
It would help a lot if you make
`static let basePath             = NSHomeDirectory() + "/Documents"` 
configurable so I can choose where to put all VMs and other resources. Like on thunderbolt external drive.

**Comment by kapsolas** [comment]
I agree. hopefully the new version, when it comes out configures this as such.

**Comment by DanzigerGeist** [comment]
Double that, I tend to keep those images in a designated folder on my system. Tried symlinking, but that did not work as the machine would not start.

**Comment by yep** [comment]
Thanks for reporting this issue. This functionality got lost during the re-write using AppKit instead of SwiftUI and has been re-added in the upcoming [version 2.1](https://github.com/yep/virtualOS/releases/tag/v2.1).
Use the app settings window to configure the VM file storage location.
